frappe/public/js/frappe/widgets/number_card_widget.js

```diff
@@ -27,26 +27,28 @@ export default class NumberCardWidget extends Widget {
 		this.make_card();
 	}
 
-	make_card() {
-		frappe.model.with_doc("Number Card", this.number_card_name || this.name).then((card) => {
-			if (!card) {
-				if (this.document_type) {
-					frappe.run_serially([
-						() => this.create_number_card(),
-						() => this.render_card(),
-					]);
+	async make_card() {
+		await frappe.model
+			.with_doc("Number Card", this.number_card_name || this.name)
+			.then((card) => {
```

Comment:
is there any reason for having both `async await` and `.then` ?

Reply:
Actually no need of `async await`. I fixed it. Thanks.